ci/scripts/python_wheel_windows_build.bat

```diff
@@ -27,15 +32,15 @@ choco install -r -y --no-progress vcredist140
 choco upgrade -r -y --no-progress vcredist140
 dir C:\Windows\System32\msvcp140.dll
 
-echo "=== (%PYTHON_VERSION%) Clear output directories and leftovers ==="
+echo "=== (%PYTHON%) Clear output directories and leftovers ==="
```

Comment:
Can we remove `setx PYTHON_VERSION` from `.dockerfile`s?

Reply:
We probably can indeed.

Reply:
Hmm, not really, because we're using it in the `RUN choco install ...` lines.

Reply:
Ah, I missed it.